rest_framework_nested/viewsets.py

```diff
@@ -44,7 +44,7 @@ def get_queryset(self):
         Filter the `QuerySet` based on its parents as defined in the
         `serializer_class.parent_lookup_kwargs` or `viewset.parent_lookup_kwargs`
         """
-        queryset = super(NestedViewSetMixin, self).get_queryset()
+        queryset = super().get_queryset()
```

Comment:
That part can stay. It helps typing tools.
```suggestion
        queryset = super(NestedViewSetMixin, self).get_queryset()
```